framer/DOMEventManager.coffee

```diff
@@ -9,21 +9,14 @@ EventManagerIdCounter = 0
 class DOMEventManagerElement extends EventEmitter
 
 	constructor: (@element) ->
-		@_elementGestureManager = new GestureManagerElement(@element)
 
 	addListener: (eventName, listener, capture=false) ->	
 		super(eventName, listener)
-		if Events.isGestureEvent eventName
-			@_elementGestureManager.addEventListener(eventName, listener, capture)
-		else
-			@element.addEventListener(eventName, listener)
+		@element.addEventListener(eventName, listener, false)
```

Comment:
Exactly. The dom removeEventListener is very picky and needs an exact reference to the handler. And if I account for them here I get nice things like `removeAllEventListeners()` for free.